src/protocols/rdp/channels/cliprdr.c

```diff
@@ -545,6 +560,7 @@ static void guac_rdp_cliprdr_channel_connected(rdpContext* context,
 
     cliprdr->MonitorReady = guac_rdp_cliprdr_monitor_ready;
     cliprdr->ServerFormatList = guac_rdp_cliprdr_format_list;
+    cliprdr->ServerFormatListResponse = guac_rdp_cliprdr_server_format_list_response;
```

Comment:
Nice, so this is where we hook into the Format List PDU
Where did you find the documentation that indicated that you need to hook into `ServerFormatListResponse` out of interest?

Reply:
I compared [This](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpeclip/395bc830-f2c2-40e5-a3f3-23e41183b777) against the clipboard implementations of FreeRDP and Guacamaole.

It was just a gut feeling once I added the trace logging you can see in the PR, in the cases where it failed, the "ctrl+v" always occurred before the "FormatListResponse".

Reply:
that's cool
I more meant where in the guac docs or somilar did you find that the `ServerFormatListResponse` variable/attribute name